src/control/dmg/utils.go

```diff
@@ -110,3 +121,23 @@ func unpackFormat(i interface{}) string {
 	out := "Listing %[1]ss on connected storage servers:\n"
 	return fmt.Sprintf("%s%s\n", out, s)
 }
+
+// getConsent scans stdin for yes/no
+func getConsent() bool {
+	var response string
+
+	_, err := fmt.Scanln(&response)
+	if err != nil {
+		fmt.Printf("Error reading input: %s\n", err)
+		return false
+	}
+
+	if response == "no" {
+		return false
+	} else if response != "yes" {
+		fmt.Println("Please type yes or no and then press enter:")
```

Comment:
It might be worth repeating the prompt if neither yes or no are specified because the prompt could get lost if enough garbage is input by accident.  Not sure if this is a practical problem, but wanted to bring it up just in case we want to worry about these things.

Reply:
good point but I'm not sure we necessarily need to worry about that in practice